src/main/java/Duke.java

```diff
@@ -1,10 +1,141 @@
+import java.util.Scanner;
+import java.util.Vector;
+
 public class Duke {
+    public class Task {
+        protected String description;
+        protected boolean isDone;
+        protected String taskType;
+        protected String taskDate;
+    
+        public Task(String description,boolean isDone, String taskType, String taskDate) { // initiating an instance
+            this.description = description;
+            this.isDone = isDone;
+            this.taskType = taskType; // todo,ddl,event
```

Comment:
I think you should standardise comments to be indented relative to their position in the code?
e.g.
while (true) {
// Do something
something();
}